git-changebar/src/gcb-plugin.c

```diff
@@ -1044,13 +1053,13 @@ goto_next_hunk_diff_hunk_cb (const git_diff_delta *delta,
       if (data->next_line >= 0) {
         return 1;
       } else if (data->line < hunk->new_start - 1) {
-        data->next_line = hunk->new_start - 1;
+        data->next_line = (hunk->new_start == 0) ? 0 : hunk->new_start - 1;
```

Comment:
maybe it'd be worth creating a function/macro for that, as it's repeated 3 times?  Not sure of a name for it yet though :)